deezerapp/src/app/app-routing.module.ts

```diff
@@ -1,11 +1,17 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+import { NgModule } from "@angular/core";
+import { Routes, RouterModule } from "@angular/router";
+import { ArtistInfoComponent } from "./artist-info/artist-info.component";
+import { ArtistSearchComponent } from "./artist-search/artist-search.component";
+import { ArtistSearchedComponent } from "./artist-searched/artist-searched.component";
 
-
-const routes: Routes = [];
+const routes: Routes = [
+  { path: "", component: ArtistSearchComponent },
+  { path: "artist/:id", component: ArtistInfoComponent },
+  { path: "search/:name", component: ArtistSearchedComponent }
+];
```

Comment:
Please also add a component to handle routes that aren't matched. i.e. `{ path: '**', component: PageNotFoundComponent}`